config/src/main/java/org/springframework/security/config/annotation/web/configurers/saml2/Saml2LoginConfigurer.java

```diff
@@ -103,6 +111,71 @@
 
 	private RelyingPartyRegistrationRepository relyingPartyRegistrationRepository;
 
+	private AuthenticationManager authenticationManager;
+	private AuthenticationConfigurationSpec<B> authenticationConfiguration;
+
+	private Saml2WebSsoAuthenticationFilter saml2WebSsoAuthenticationFilter;
+
+	/**
+	 * Allows a configuration of a {@link AuthenticationManager} to be used during SAML 2 authentication.
+	 * If none is specified, the system will create one inject it into the {@link Saml2WebSsoAuthenticationFilter}
+	 * @param authenticationManager the authentication manager to be used
+	 * @return the {@link Saml2LoginConfigurer} for further configuration
+	 * @throws IllegalArgumentException if authenticationManager is null
+	 * @throws IllegalArgumentException if {@link #authenticationConfiguration} has already been invoked to
+	 * 									configure the default manager
+	 * @since 5.2.2
```

Comment:
Because these are changes to the public API and are not addressing a bug, we should [target a minor release instead](https://semver.org/#spec-item-7).

Reply:
Do we hold back additive changes? Aren't those simply backwards compatible?

Reply:
Even for additive APIs? Rather than simply pushing out the code and let it be used?

Reply:
I misunderstood. Updated to 5.3